src/code/m_lights.c

```diff
@@ -0,0 +1,366 @@
+#include "global.h"
+#include "m_lights.h"
+#include "gfx.h"
+#include "macros.h"
+#include "sys_matrix.h"
+#include "overlays/gamestates/ovl_play/m_play.h"
+#include "6E0F50.h"
+
+extern Gfx D_400AA00[];
+extern Gfx D_400AA40[];
+
+static LightBuffer light_list_buf;
```

Comment:
why the `static`?

Reply:
Because it's not used outside the file?

Reply:
But this is bss and you know we avoid using `static` on bss symbols when possible